### PR TITLE
Add bg color option to form containers

### PR DESCRIPTION
**Pull request by elpaso**
Because gray is boring.

peek 24-02-2019 13-04

### CODE REVIEW

**Comment by nirvn** [comment]
And because we _could_, why not adding a data-defined button and have the possibility of modifying the background color value based on an expression? :smile: That could be useful.

**Comment by elpaso** [comment]
@nirvn yes, nice idea! Something for the future.

File: src/app/qgsattributesformproperties.cpp [hunk 1]
```diff
@@ -1046,6 +1050,17 @@ void DnDTree::onItemDoubleClicked( QTreeWidgetItem *item, int column )
         layout->addRow( showAsGroupBox );
       }
 
+      QgsCollapsibleGroupBox *styleGroupBox = new QgsCollapsibleGroupBox( tr( "Style" ), layout->widget() );
+      styleGroupBox->setObjectName( QLatin1Literal( "attributeFormPropertiesContainerStyle" ) );
+      QFormLayout *customizeGroupBoxLayout = new QFormLayout( styleGroupBox ) ;
+      QgsColorButton *backgroundColorButton = new QgsColorButton( styleGroupBox, tr( "Select the background color for the container" ) );
```
Comment (nyalldawson) [inline src/app/qgsattributesformproperties.cpp]: I think the title here should just be "Container Background Color" for consistency with other color dialogs

**Review by nyalldawson — COMMENTED** [review]
Nice! Mind adding some unit tests here too? (Just covering serialisation/deserialisation of the color, not the actual stylesheet stuff itself)

**Pull request merged**